src/reportWebVitals.js

```diff
@@ -1,6 +1,6 @@
-const reportWebVitals = onPerfEntry => {
+const reportWebVitals = (onPerfEntry) => {
```

Comment:
ooh are we measuring app metrics?

Reply:
not planned at the moment, this was just made by create react app 😅